app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
```

Comment:
remove device all the files if you are not overriding any of the method
same for the view